core/src/main/scala/com/codacy/analysis/core/analysis/CodacyPluginsAnalyser.scala

```diff
@@ -51,6 +52,25 @@ class CodacyPluginsAnalyser extends Analyser[Try] {
     result.map(_.to[Set])
   }
 
+  override def duplication(duplicationTool: DuplicationTool,
+                           directory: File,
+                           files: Set[Path],
+                           timeout: Option[Duration] = Option.empty[Duration]): Try[Set[DuplicationClone]] = {
+
+    val srcFiles = FilesTarget(directory, files, Set.empty[Path])
+
+    val result = duplicationTool.run(directory, srcFiles, timeout)
```

Comment:
If we need a `FilesTarget` here why would we pass only the files and then re-create the object?

Reply:
For the Analyser[T] (and consequently CodacyPluginsAnalyser) to have consistent signatures. I can settle of FilesTarget for all of them or I can recreate here.